substrate/runtime/staking/src/lib.rs

```diff
@@ -78,23 +75,26 @@ pub enum LockStatus<BlockNumber: Parameter> {
 	LockedUntil(BlockNumber),
 	Bonded,
 }
-
+/*
 /// Preference of what happens on a slash event.
 #[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
 #[derive(Encode, Decode, Eq, PartialEq, Clone, Copy)]
-pub struct SlashPreference {
+pub struct ValidatorPrefs<Balance: Parameter + Codec + MaybeSerializeDebug + MaybeDeserialize> {
```

Comment:
For the reference: Why is this commented?

Reply:
I want to bring this in soon, but right now i can't make it work.